### PR TITLE
fix multiple flows on the same page

### PR DESCRIPTION
**Pull request by nirgur**

### CODE REVIEW

File: packages/sdks/web-component/src/lib/helpers/helpers.ts [hunk 1]
```diff
@@ -109,10 +114,17 @@ export function getAnimationDirection(
...
 }
 
-export const getRunIdsFromUrl = () => {
-  const [executionId = '', stepId = ''] = (getFlowUrlParam() || '').split('_');
+export const getRunIdsFromUrl = (flowId: string) => {
+  let [executionId = '', stepId = ''] = (getFlowUrlParam() || '').split('_');
+  const executionFlowId = getFlowIdFromExecId(executionId);
+
+  // if the flow id does not match, this execution id is not for this flow
+  if (executionFlowId && executionFlowId !== flowId) {
```
Comment (dorsha) [inline packages/sdks/web-component/src/lib/helpers/helpers.ts:122]: Let's also add if `flowId` exists

File: packages/sdks/web-component/test/helpers/index.test.ts [hunk 2]
```diff
@@ -59,8 +59,12 @@ describe('helpers', () => {
       writable: true,
       value: new URL('http://localhost'),
     });
-    window.location.search = `?${URL_RUN_IDS_PARAM_NAME}=8_9`;
-    expect(getRunIdsFromUrl()).toEqual({ executionId: '8', stepId: '9' });
+    window.location.search = `?${URL_RUN_IDS_PARAM_NAME}=8|#|a_9`;
+    expect(getRunIdsFromUrl('8')).toEqual({
```
Comment (dorsha) [inline packages/sdks/web-component/test/helpers/index.test.ts:63]: let's also add test in case of empty flow id, and cases with wrong location.search query param

File: packages/sdks/web-component/src/lib/helpers/helpers.ts [hunk 3]
```diff
@@ -57,6 +57,11 @@ function resetUrlParam(paramName: string) {
   }
 }
 
+const getFlowIdFromExecId = (executionId: string) => {
+  const regex = /(.*)\|#\|.*/;
+  return regex.exec(executionId)?.[1];
```
Comment (dorsha) [inline packages/sdks/web-component/src/lib/helpers/helpers.ts:62]: can it fail if length is 0 or 1?

File: packages/sdks/web-component/test/helpers/index.test.ts (hunk 2 above)
Comment (nirgur) [reply packages/sdks/web-component/test/helpers/index.test.ts:63]: done

File: packages/sdks/web-component/src/lib/helpers/helpers.ts (hunk 1 above)
Comment (nirgur) [reply packages/sdks/web-component/src/lib/helpers/helpers.ts:122]: AFAIR if there is no flow id there is an error and the SDK is not running
Added it anyway

File: packages/sdks/web-component/src/lib/helpers/helpers.ts (hunk 3 above)
Comment (nirgur) [reply packages/sdks/web-component/src/lib/helpers/helpers.ts:62]: No, the regex.exec return an array or null
Changed to return an empty string in this case

**Review by dorsha — APPROVED** [review]
